plugins/inputs/lotus/daemon.go

```diff
@@ -67,7 +67,7 @@ func NewDaemon(addr string, token string) (*Daemon, error) {
 	var daemonApi lotusapi.FullNodeStruct
 	_, err := jsonrpc.NewMergeClient(context.Background(), "ws://"+addr+"/rpc/v0", "Filecoin", []interface{}{&daemonApi.Internal, &daemonApi.CommonStruct.Internal}, headers)
 	if err != nil {
-		log.Printf("addr: %s, token: %s", addr, token)
```

Comment:
Maybe we should put something more descriptive such as `"Connecting to addr: %s`

Reply:
Or  actually probably 1 merged message would be enough:
```go
log.Fatalf("failed to connect to lotus daemon at %s with error: %s", addr, err)
```